ui/main.qml

```diff
@@ -42,8 +42,8 @@ StatusWindow {
     objectName: "mainWindow"
     minimumWidth: 900
     minimumHeight: 600
-    width: 1232
-    height: 770
+    width: Math.min(1232, Screen.desktopAvailableWidth - 64)
```

Comment:
this is not strictly related to the issue, but it was so annoying on my laptop (4K @ 250% scale) - the window spans beyond my desktop area.